cypress/integration/pipeline.spec.ts

```diff
@@ -126,9 +127,9 @@ describe('Creating a pipeline', () => {
 
     cy.contains('error found').should('exist');
 
-    // Fix missing field to resolve error
-    cy.get('[data-cy="message"]').within(() => {
-      cy.get('button').click();
+    cy.get('[data-cy="message"] textarea').type(TEST_EMAIL_MACRO, {
```

Comment:
Nit: the comment is still valid. The test triggers an empty field error before validating correctly